web/src/ee/features/billing/server/cloudBillingRouter.ts

```diff
@@ -0,0 +1,257 @@
+import { createStripeClientReference } from "@/src/ee/features/billing/stripeClientReference";
+import { stripeClient } from "@/src/ee/features/billing/utils/stripe";
+import { stripeProducts } from "@/src/ee/features/billing/utils/stripeProducts";
+import { env } from "@/src/env.mjs";
+import { throwIfNoEntitlement } from "@/src/features/entitlements/server/hasEntitlement";
+import { parseDbOrg } from "@langfuse/shared";
+import {
+  createTRPCRouter,
+  protectedOrganizationProcedure,
+} from "@/src/server/api/trpc";
+import { TRPCError } from "@trpc/server";
+import * as z from "zod";
+import { throwIfNoOrganizationAccess } from "@/src/features/rbac/utils/checkOrganizationAccess";
+
+export const cloudBillingRouter = createTRPCRouter({
```

Comment:
I dont see any audit logs here. I think we should add these.

Reply:
done for create checkout session